### PR TITLE
BACKUP GROUPS: documentation

### PR DESCRIPTION
**Pull request by ethouris**

### CODE REVIEW

File: docs/API-functions.md [hunk 1]
```diff
@@ -557,6 +558,14 @@ where:
 * `result`: result of the operation (if this operation recently updated this structure)
 * `srcaddr`: address to which `id` should be bound
 * `peeraddr`: address to which `id` should be connected
+* `priority`: priority for backup group
+
+The priority is set to 0 by default by `srt_prepare_endpoint()` - you can set
+it to a different value afterwards. The default 0 value is the highest priority
+and greater values declare lower priorities. The priority for the backup
+groups decide which link to activate first when the currently active link is
```
Comment (stevomatthews) [inline docs/API-functions.md]: groups determines which link is activated first when the currently active link is

File: docs/API-functions.md [hunk 2]
```diff
@@ -557,6 +558,14 @@ where:
...
+unstable and which should keep transmitting when multiple active links are
```
Comment (stevomatthews) [inline docs/API-functions.md]: groups determines which link is activated first when the currently active link is
unstable, and which should keep transmitting when multiple active links are

File: docs/API.md [hunk 3]
```diff
@@ -530,6 +530,27 @@ be set on a socket.
...
+two consecutive packets (of whatever type, but usually this is about control
+packets while the agent is a sender) retrieved from the peer on the currently
+active link (note that idle links exchange only keepalive messages once per 1s,
+so they do not count). This value should be set as small as possible, although
+note that 10ms is the time between two consecutive ACK messages, and various
+deviations from this value will happen, hence 40ms is considered most recommended
+value. Note that the higher this time is, the higher latency penalty you get.
+When this time is exceeded and the link is therefore considered unstable, one
+of the idle links gets activated, and then all packets unacknowledged so far
+are first sent.
```
Comment (stevomatthews) [inline docs/API.md]: - This setting is used for groups of type `SRT_GTYPE_BACKUP`. It defines the stability 
timeout, which is the maximum interval between two consecutive packets retrieved from 
the peer on the currently active link.
  
NOTE: The two packets can be of any type, but this setting usually refers to control
packets while the agent is a sender. Idle links exchange only keepalive messages once 
per second, so they do not count. 

The value of `SRTO_GROUPSTABTIMEO` should be set as small as possible. Keep in mind that 
the time between two consecutive ACK messages (default = 10 ms) will vary, hence 40ms is 
the recommended setting. Note that higher values increase the latency penalty. When this 
time is exceeded, and the link is therefore considered unstable, one of the idle links 
gets activated, at which point all packets unacknowledged so far are sent first.

**Comment by ethouris** [comment]
I made some formatting changes towards the fixed text, although the contents should be the same.

File: docs/bonding-intro.md [hunk 4]
```diff
@@ -1,5 +1,4 @@
-What are groups
-===============
+# What are groups #
 
 A Group is an entity that binds multiple sockets and it is required to
```
Comment (maxsharabayko) [inline docs/bonding-intro.md]: No closing `#` is needed. Just as simple as
`# What are groups`

Comment (ethouris) [reply docs/bonding-intro.md]: But allowed... :)

Comment (maxsharabayko) [reply docs/bonding-intro.md]: Indeed. Then put a thousand of them. :)

**Pull request merged**